src/CameraScreen/style.ts

```diff
@@ -0,0 +1,49 @@
+import { StyleSheet } from "react-native";
```

Comment:
Not a ton new in this file, it just got moved from the previously named one. Some centering and positioning got updated.